docs/spec/api.md

```diff
@@ -1,12 +1,8 @@
-<!--[metadata]>
-+++
-title = "HTTP API V2"
-description = "Specification for the Registry API."
-keywords = ["registry, on-prem, images, tags, repository, distribution, api, advanced"]
-[menu.main]
-parent="smn_registry_ref"
-+++
-<![end-metadata]-->
+---
+title: "HTTP API V2"
+description: "Specification for the Registry API."
+keywords: ["registry, on-prem, images, tags, repository, distribution, api, advanced"]
+---
```

Comment:
This probably needs to be updated here as well; https://github.com/docker/distribution/blob/master/docs/spec/api.md.tmpl#L1-L9

(or instead: @stevvooe ?)